### DISCUSSION TITLE
cannot read property uidt of undefined [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2867] [Discord Threads]
> 🙋‍♂️ **ybon**  
#### Any clue ? Thanks 🙂  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/999244861611974706/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### pls create an issue n share a sample excel file for reproducing the issue thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999244862283071488)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
